tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/CopyContent.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.tools.contentgenerator.cli;
+
+import jakarta.validation.constraints.Min;
+import java.util.ArrayList;
+import java.util.List;
+import org.projectnessie.client.api.NessieApiV2;
+import org.projectnessie.error.BaseNessieClientServerException;
+import org.projectnessie.model.Content;
+import org.projectnessie.model.ContentKey;
+import org.projectnessie.model.Operation.Put;
+import org.projectnessie.model.Reference;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "copy", mixinStandardHelpOptions = true, description = "Repeatedly copy content")
+public class CopyContent extends CommittingCommand {
+
+  @Option(
+      names = {"-r", "--ref", "--branch"},
+      description = "Branch name for making changes (defaults to the default branch if not set).")
+  private String ref;
+
+  @Min(value = 1, message = "At least one copy is to be made.")
+  @Option(
+      names = {"-n", "--num-copies"},
+      required = true,
+      defaultValue = "10000",
```

Comment:
isn't this a bit extreme for a default value?

Reply:
This tools is not really meaningful aside from stress tests, so I'd say it's right on target ;)